packages/cycle-scripts-one-fits-all/configs/babelrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "presets": [
+        [ "env", {
+            "targets": {
+                "browsers": ["last 2 versions"],
+                "uglify": true
```

Comment:
Yeah, i will merge it with babelrc. It does not mean that it will be uglified, but this line will go away anyway